build-core-riscv64-spike.sh

```diff
@@ -32,6 +32,10 @@ make -C "phoenix-rtos-devices" all install
 b_log "Building coreutils"
 make -C "phoenix-rtos-utils" all install
 
+b_log "Building hostutils"
+make -C "phoenix-rtos-hostutils" -f Makefile.old all
```

Comment:
Why Makefile.old?

Reply:
New `Makefile` generates compilation errors. On the other platforms `Makefile.old` is used as well.
I haven't gone into details of it.